### ISSUE TITLE
ImportError: cannot import name 'constant'

### ISSUE THREAD

**Issue by yanzmgit** [vijos/vj4#415]
When I install the vj4， npm build succeds. But it  fails when I run it ,  the errors are as：

[root@yan vj4]# python3 -m vj4.server --debug --listen http://0.0.0.0:8888     
Traceback (most recent call last):  
  File "/usr/local/lib/python3.6/runpy.py", line 193, in _run_module_as_main  
    "__main__", mod_spec)  
  File "/usr/local/lib/python3.6/runpy.py", line 85, in _run_code  
    exec(code, run_globals)  
  File "/root/downloads/yan/vj4/vj4/server.py", line 12, in <module>  
    from vj4 import app  
  File "/root/downloads/yan/vj4/vj4/app.py", line 9, in <module>  
    from vj4 import error  
  File "/root/downloads/yan/vj4/vj4/error.py", line 1, in <module>  
    from vj4.model import builtin  
  File "/root/downloads/yan/vj4/vj4/model/builtin.py", line 6, in <module>  
    from vj4 import constant  
ImportError: cannot import name 'constant'

I checked  the  vj4/model/builtin.py， I find the wrong code is  “from vj4 import constant”.But there is so constant python packege in the project,  expect  a directory located in the vj4/ui which includes some js files。

Forgive my ignorance， How can python import js files？ and how can i  fix this problem. Thanks.

**Comment by breeswish** [comment]
Please run `npm run build` first, which will generate `vj4.constant` from JavaScript files.

**Comment by yanzmgit** [comment]
Thankful for your reply. @breeswish 
But as I wrote before,   npm run build shoud have been done. And I tried once more, and the outputs is:
[root@yan vj4]# npm run build
> vj4@4.0.0 build /root/downloads/yan/vj4
> node scripts/build
[root@yan vj4]# echo $?
0

Still reports the error  " cannot import name 'constant'" when I run "python3 -m vj4.server --debug --listen http://0.0.0.0:8888".

could it be  possible that this build process  are not successful?
